celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -423,55 +121,68 @@ def runner(tech_matrix, F,i,l,j,k,route_id,final_demand_scaler):
         """
         tim0 = time.time()
         res = pd.DataFrame()
-        res2 = pd.DataFrame()
-        res,res2 = solver_optimization(tech_matrix, F)
+        res = solver(tech_matrix, F,process_emissions)
         res['value'] = res['value']*final_demand_scaler
         if not res.empty:
           res.loc[:,'year'] =  i
           res.loc[:,'facility_id'] =  l
           res.loc[:,'stage'] = j
           res.loc[:,'material'] = k
-          res.loc[:, 'route_id'] = route_id
+          res.loc[:,'route_id'] = route_id
     
-          print(str(i) +' - '+j + ' - ' + k)
+          print(str(i) +' - '+j + ' - ' + k, flush = True)
```

Comment:
Do we need this print statement? Looks like there's something printed if the calculations work and something printed if they fail.

Reply:
I kept this print statement in order to track the solutions as they occur in real time. I will say once we see that code is running perfectly on exploring the results, we can delete this line.

Reply:
Verbose parameter added.